### PR TITLE
 update process-obs/FV3/gbbepx2netcdf/mk-hera.sh for Rocky8 OS

### PR DESCRIPTION
**Pull request by zhanglikate**
Update compile script with module load intel/2023.2.0 for Rocky OS

### CODE REVIEW

**Review by SamuelTrahanNOAA — CHANGES_REQUESTED** [review]
This doesn't work for me. I get this error:

```
Lmod has detected the following error:  Cannot load module "intel/2023.2.0". At least one of these module(s) must be loaded:
   gnu

While processing the following module(s):
    Module fullname  Module Filename
    ---------------  ---------------
    intel/2023.2.0   /apps/modules/modulefiles/intel/2023.2.0
```

**Comment by zhanglikate** [comment]
Hi Sam, 

      I am using the following build script . there is not issue. 

#! /usr/bin/env bash
set -eux


cd ./prepchem_NC.fd/process-obs/FV3/gbbepx2netcdf/

echo " Building ... prepchem to convert binary to NetCDF"

sh mk-hera.sh
cp mkncgbbepx.exe ../../../../../exec/mkncgbbepx

**Comment by zhanglikate** [comment]
This is the mk-hera.sh

set -ue
module load intel/2023.2.0 netcdf szip hdf5
set -x

fflags=$( nf-config --fflags )
flibs=$( nf-config --flibs )

ifort $fflags -I. -c -o mkncgbbepx.o mkncgbbepx.f90 -g -traceback
ifort $fflags -o mkncgbbepx.exe mkncgbbepx.o -g -traceback $flibs

**Comment by zhanglikate** [comment]
Sam,

     If you are testing it in your terminal , you have to use 
  
    module load gnu intel/2023.2.0 netcdf szip hdf5

**Comment by SamuelTrahanNOAA** [comment]
I still get the error with that script.

Please try purging your modules first:

```
module purge
```

File: process-obs/FV3/gbbepx2netcdf/mk-hera.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #! /bin/sh
 
 set -ue
-module load intel/19.0.5.281 netcdf szip hdf5
+module load intel/2023.2.0 netcdf szip hdf5
```
Comment (SamuelTrahanNOAA) [inline process-obs/FV3/gbbepx2netcdf/mk-hera.sh]:
```suggestion
module purge
module load gnu intel/2023.2.0 netcdf szip hdf5
module list
```

**Review by SamuelTrahanNOAA — CHANGES_REQUESTED** [review]
It's probably best to start with a clean set of modulefiles.

**Comment by zhanglikate** [comment]
Updated. Can you help to check and approve it? Thanks.

**Review by SamuelTrahanNOAA — APPROVED** [review]
This work receives the Sam Seal of Approval.

**Comment by zhanglikate** [comment]
Thanks very much.

**Pull request merged**